src/thunderbots/software/util/canvas_messenger/canvas_messenger.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "thunderbots_msgs/CanvasLayer.h"
+
+// Forward declaration
+namespace ros
+{
+    class NodeHandle;
+    class Publisher;
+}  // namespace ros
+
+namespace Util
+{
+    using CanvasLayer = thunderbots_msgs::CanvasLayer;
```

Comment:
`using` declarations should never be in the header like this, because if anyone else includes the header, they now "get" the declaration as well. Basically we pretty much always have to use the full declaration in the header (though this can totally be moved to the `.cpp`, and any usages there can remain unchanged!)

Same goes for the other `using` declarations below.

Reply:
Removed